package.json

```diff
@@ -76,5 +77,6 @@
     "webpack-cli": "^3.3.1",
     "yargs": "^13.2.2"
   },
-  "keywords": []
+  "keywords": [],
+  "dependencies": {}
```

Comment:
Why did you add this?

Reply:
It's probably when I added https://www.npmjs.com/package/@types/webaudioapi.
I am going to see if it works without it first.

Reply:
It works without it.

Reply:
Done.